packages/babel-preset-expo/lazy-imports-blacklist.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * These expo packages may have side-effects and should not be lazy-initialized.
+ */
+'use strict';
+
+module.exports = new Set([
```

Comment:
This is a large set of packages -- let's check each one to see if the side effects actually exist and are important.

Reply:
I've checked each package and removed the false positives.